src/backend/src/services/projects.services.ts

```diff
@@ -751,13 +751,10 @@ export default class ProjectsService {
 
     if (manufacturer.dateDeleted) throw new DeletedException('Manufacturer', manufacturer.name);
```

Comment:
no need to check for this anymore because the manufacturers are no longer being soft deleted